events/observation_import_complete.go

```diff
@@ -0,0 +1,23 @@
+package events
+
+import "github.com/ONSdigital/go-ns/avro"
+
+// ObservationImportCompleteEvent is used to notify that all observations of an instance have been imported.
```

Comment:
Doc doesnt match type name

Reply:
This event was redundant after the design update. I have removed it.